tstyche.config.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://tstyche.org/schemas/config.json",
   "checkSuppressedErrors": true,
+  "checkSourceFiles": false,
```

Comment:
Much better performance when this is disabled. The next major will not have this option, the implementation is not good. It simply type-checks all files I the program. Each and every `.ts` file. That is already done by `tsc`. So.. That was bad idea ;D